espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Http/Response/AppiumResponse.java

```diff
@@ -1,27 +1,62 @@
 package io.appium.espressoserver.lib.Http.Response;
 
-import io.appium.espressoserver.lib.Model.Appium;
+import java.util.UUID;
+
+import fi.iki.elonen.NanoHTTPD.Response.Status;
 import io.appium.espressoserver.lib.Model.AppiumStatus;
 
-public class AppiumResponse extends BaseResponse {
-    public AppiumResponse() {
-        setResponse(new Appium());
+@SuppressWarnings("unused")
+public class AppiumResponse {
+    private transient Status httpStatus;
+    private Object value;
+    private AppiumStatus status;
+    private String sessionId;
+    private String id; // Unique Appium transaction ID
+
+    AppiumResponse(AppiumStatus status, Object value) {
+        init(status, value, null);
+    }
+
+    public AppiumResponse(AppiumStatus status, Object value, String sessionId) {
+        init(status, value, sessionId);
+    }
+
+    private void init(AppiumStatus status, Object value, String sessionId) {
```

Comment:
one can simply [call one constructor from another](https://stackoverflow.com/questions/285177/how-do-i-call-one-constructor-from-another-in-java). No need to use a separate method for this purpose.